src/app/components/Curation/CurationPromo/index.tsx

```diff
@@ -44,9 +44,17 @@ const CurationPromo = ({
     (type === 'video' && `${videoTranslation}, `) ||
     (type === 'photogallery' && `${photoGalleryTranslation}, `);
 
+  const promoImage = imageUrl || '';
+  const promoImageAltText = imageUrl && (imageAlt || 'image');
```

Comment:
Hmmm I am wondering whether we should do this or not. The reason it's happening is because some of the images on the Kyrgyz test page do not have alt text - https://www.test.bbc.com/kyrgyz - many of the images are the default BBC News ones e.g. https://ichef.bbci.co.uk/ace/ws/240/cpsprodpb/36D1/production/_127933041__63970643_bbc-news-world-service-logo-nc.png

In this PR, we have updated the fixture data to be in line with what's currently on test. If we look at the diff, it appears that some alt text was manually added, but we want to avoid that going forward, as it is a maintenance nightmare.

In Tipo, users must include alt text for any images which are uploaded, so we didn't think that we needed to enforce it here. 

Ideally, the fallback alt text would come from the BFF, rather than being coded here. 

So, I am wondering if instead, we revert these changes, and test a different homepage in the lighthouse tests - ideally one which has images with alt text?